frontend/src/store/app/actions.ts

```diff
@@ -18,4 +19,15 @@ const getTenant = createAsyncThunk<
   return tenant;
 });
 
-export { getTenant };
+const updateTenant = createAsyncThunk<
+  EAMTenantByIdResponseDto,
+  EAMTenantUpdateRequestDto,
+  AsyncThunkConfig
+>(ActionType.UPDATE_TENANT_NAME, async (payload, { extra }) => {
+  const { tenantApi, notification } = extra;
+  const tenant = await tenantApi.updateTenant(payload);
+  notification.success('Success!', 'Tenant has been successfully created');
```

Comment:
we have enums for it. use them

Reply:
What do you mean?, I don't quite understand, tell me please

Reply:
thanks)